submissions/maryOkosun/phpDatabase/viewQuery.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+include("cookie2.php");
```

Comment:
The `$sql` query will always run each time this page is reloaded on the browser since the `cookie2.php` is `include` here. The idea is to only run that query as long as cookie is not set